pkg/kubelet/remote/remote_runtime.go

```diff
@@ -151,7 +155,9 @@ func (r *RemoteRuntimeService) CreateContainer(podSandBoxID string, config *runt
 	ctx, cancel := getContextWithTimeout(r.timeout)
 	defer cancel()
 
+	uid := uuid.NewUUID()
```

Comment:
Is this the "attempt" ID, really?  Using the pod UID is correct because it is reproducible.  We can crash without fear of creating a duplicate pod.  We don't have a UID per-container, so if this crashed halfway through, are we SURE that we would not send a duplicate CreateContainer() call with a new random UUID?